src/idl_gen_cpp.cpp

```diff
@@ -446,7 +446,7 @@ class CppGenerator : public BaseGenerator {
   // Return a C++ type from the table in idl.h
   std::string GenTypeBasic(const Type &type, bool user_facing_type) const {
     static const char *ctypename[] = {
-    // clang-format off
```

Comment:
why were these spaces removed?

Reply:
oh! looks like a merge error. will fix this in the next rebase.